test/modelnode.jl

```diff
@@ -338,3 +338,34 @@ end
         end
     end
 end
+
+gradients_match(grads1::Flux.Zygote.Grads, grads2::Flux.Zygote.Grads) = grads1.params == grads2.params && gradients_match(grads1.grads, grads2.grads)
+gradients_match(grads1::Flux.Zygote.Grads, grads2::Flux.Zygote.Grads) = grads1.params == grads2.params && gradients_match(grads1.grads, grads2.grads)
```

Comment:
In my humble opinion, `using Flux: Zygote.Grads, onehot, onehotbatch` in `test/runtests.jl` is slightly prettier (or get rid of just `Flux`).

Reply:
ah, yes, that does look better